plugins/module_utils/redfish_utils.py

```diff
@@ -3616,7 +3616,7 @@ def set_session_service(self, sessions_config):
 
     def verify_bios_attributes(self, bios_attributes):
         # This method verifies BIOS attributes against the provided input
-        server_bios = self.get_multi_bios_attributes()
+        server_bios = self.get_bios_attributes(self.systems_uri)
```

Comment:
Correct (that was the case before my change as well):
```
msg: Issuing a data modification command without specifying the ID of the target System resource when there is more than one System is no longer allowed. Use the `resource_id` option to specify the target System ID.
```

That threw me off, but because everything in `redfish_command` uses `data_modification=True`:

https://github.com/ansible-collections/community.general/blob/baef8580ecb293af527340bd806a8f31f0d42d30/plugins/modules/redfish_command.py#L958

it hits the gate:
https://github.com/ansible-collections/community.general/blob/a1c39cc8820efa6c52658065d5b9c5c4c89ee0ee/plugins/module_utils/redfish_utils.py#L518-L519

Reply:
Okay, just checking. I don't think it was ever reasonable in the first place to have a "multi-system" BIOS check; I think it would be very likely there'd be differences. I prefer having this check a specific system.